venvUtils/ensureVenv.py

```diff
@@ -25,24 +25,26 @@
 isInteractive = hasattr(sys.stdout, "isatty") and sys.stdout.isatty()
 
 
-def askYesNoQuestion(message: str, default: bool = True) -> bool:
+def askYesNoQuestion(message: str) -> bool:
 	"""
 	Displays the given message to the user and accepts y or n as input.
 	Any other input causes the question to be asked again.
-	If isInteractive is False, the default is always returned.
+	If isInteractive is False, y is assumed and True is returned.
 	@returns: True for y and False for n.
 	"""

```

Comment:
This is still implicit.

Instead:
```python
def askYesNoQuestion(message: str, default: bool) -> bool:
	"""
	Displays the given message to the user and accepts y or n as input.
	Any other input causes the question to be asked again.
	If isInteractive is False, the default is always returned, the question and outcome will still be sent to stdout for inspection of the build.
	@param default: the return value when the user can not be prompted.
	@returns: True for y and False for n.
	"""
```

Reply:
Ah I interpreted explicit the other way around 🤣